source/hierarchy.txt

```diff
@@ -251,3 +444,14 @@ These examples also apply other recommended configurations, including:
       see |service-terraform| and the `MongoDB Terraform Blog Post
       <https://www.mongodb.com/developer/products/atlas/deploy-mongodb-atlas-terraform-aws/>`__.
 
+Next Steps
+----------
+
+After you plan the hierarchy and size for your organizations, projects,
+and {+clusters+}, see the following suggested resources or use the left
+navigation to find features and best practices for each pillar.
```

Comment:
It's unclear to me what "pillar" refers to here. Does it refer to the following three links, or something else?

Reply:
Good call, pillars are the "Well-Architected Framework" categories that are the top-level left nav for these docs. I'll clarify by adding a link (currently the link just goes to mongodb.com, but it will point to the Well-Architected Framework pillars overview page when it's ready)